src/main/java/seedu/address/logic/commands/MarkAttendanceCommand.java

```diff
@@ -40,7 +40,7 @@ public class MarkAttendanceCommand extends AttendanceMarkingCommand {
             + PREFIX_TELEGRAM + "berniceYu " + PREFIX_DATE + "2024-10-21";
 
     public static final String MESSAGE_MARK_MEMBER_SUCCESS =
-            "Successfully mark the attendance on %1$s for member(s): %2$s";
+            "Successfully mark the attendance on %1$s for member(s): \n%2$s\n";
```

Comment:
Very minor nitpick would be to change `mark` to `marked`, haha

Reply:
Thanks, already changed.